app/tests/test_anti_virus_check.py

```diff
@@ -0,0 +1,127 @@
+import unittest
+
+import requests
+import requests_mock
+from sdc.rabbit.exceptions import QuarantinableError, RetryableError, BadMessageError
+
+from app import settings
+from app.anti_virus_check import AntiVirusCheck
+from app.main import Payload
+
+session = requests.Session()
+adapter = requests_mock.Adapter()
+session.mount('mock', adapter)
+
+
+class AntiVirusCheckTests(unittest.TestCase):
+
+    @requests_mock.mock()
```

Comment:
I'd strongly advise against the use of this library. It isn't very well supported and has been deprecated by OpenStack. Sentry have a viable alternative that is fully supported https://github.com/getsentry/responses